jcl/src/java.base/share/classes/openj9/internal/foreign/abi/InternalDowncallHandler.java

```diff
@@ -764,7 +764,13 @@ Object runNativeMethod(Addressable downcallAddr, SegmentAllocator segmtAllocator
 		/*[IF JAVA_SPEC_VERSION >= 21]*/
 		try (Arena arena = Arena.ofConfined()) {
 			SetDependency(arena.scope());
-			returnVal = invokeNative(linkerOpts.isTrivial(), getValidDowncallMemAddr(stateSegmt), retMemAddr, getValidDowncallMemAddr(downcallAddr), cifNativeThunkAddr, args);
+			returnVal = invokeNative(
+		/*[IF JAVA_SPEC_VERSION >= 22]*/
+					linkerOpts.isCritical(),
+		/*[ELSE] JAVA_SPEC_VERSION >= 22 */
+					linkerOpts.isTrivial(),
+		/*[ENDIF] JAVA_SPEC_VERSION >= 22 */
+					getValidDowncallMemAddr(stateSegmt), retMemAddr, getValidDowncallMemAddr(downcallAddr), cifNativeThunkAddr, args);
```

Comment:
Since the arguments are no longer on one line, suggest one per line.

Reply:
Updated